hack/rapid-deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -ex
+
+CGO_ENABLED=0 GOOS=linux go build -mod=vendor -ldflags="-X 'main.Version=develop' -buildid=" -trimpath -o linux_controller ./cmd/controller/...
+
+echo "# -- this file is autogenerated by rapid-deploy.sh from main Dockerfile and is not version controlled" > Dockerfile.dev
+
+run_image_start=`cat Dockerfile | grep -n "\- run \-" | cut -d':' -f1`
+tail -n +$run_image_start Dockerfile | \
+  sed 's/COPY.*secretgen-controller/COPY linux_controller secretgen-controller/' >> Dockerfile.dev
+
+ytt -f config/ -f config-rapid-deploy/ | kbld -f- | kapp deploy -a sg -f- -c -y
```

Comment:
Can we maybe call this `config-dev-deploy` instead?

Reply:
cool --  it started life as "dev-deploy" and then i thought,  "wait aren't _all_ the deploys that i do 'dev'?" so i went with "rapid" ... happy to change it back, seems DH & DK agree on the name.